[Round1] Step2 - Moss

0-jerry opened the pull request:
@lxodud
궁금했던 점이 있습니다.
순서도에서 printIceCream(layerOfIceCream: iceCream)가 끝나고
 printCone(layerOfStick: stick) 함수가 호출되는 경우
제가 표현한대로 하는게 맞나요?

0-jerry committed:
아이스크림 그리기

0-jerry committed:
아이스크림 그리기

0-jerry committed:
아이스크림 그리기

0-jerry committed:
아이스크림 그리기

lxodud reviewed (commented):
@lichen0423
안녕하세요 Moss!!
고생하셨습니다 👍👍👍

## 궁금했던 점
> 순서도에서 printIceCream(layerOfIceCream: iceCream)가 끝나고 printCone(layerOfStick: stick) 함수가 호출되는 경우 제가 표현한대로 하는게 맞나요?

두 함수가 순차적으로 호출되는 순서를 잘 표현해주신 것 같습니다.

현재 프로젝트 파일, swift 파일이 여러개 존재하는데 하나의 프로젝트로 정리하면 좋을 것 같습니다.
이해가 안되거나 질문이 있다면 언제든지 디엠 보내주세요 🔥🔥🔥

0-jerry committed:
Update main.swift

0-jerry committed:
Delete IceCream/print-ice-cream.drawio

0-jerry committed:
Delete [Step.2] 아이스크림 그리기.swift

0-jerry committed:
순서도 수정

0-jerry committed:
순서도 그림 추가

0-jerry committed:
순서도 그림 수정

0-jerry committed:
Delete IceCream/[Step2] 아이스크림 그리기.xcodeproj directory

0-jerry committed:
Delete IceCream directory

0-jerry committed:
아이스크림 그리기 수정

피드백을 토대로 함수의 이름을 수정해봤습니다.
그리고 중첩함수가 불필요하다고도 느껴져서 for-in 구문의 상태로 넣었습니다.

손잡이의 프린트는 고정값으로 두는게 더 일반적이라는 생각이 들어서 stick이라는 상수를 정의해 문자열을 할당해 출력했습니다.

lxodud reviewed (approved):
고생하셨습니다 👍👍👍
